pkg/pod/creds_init.go

```diff
@@ -73,6 +73,9 @@ func credsInit(ctx context.Context, serviceAccountName, namespace string, kubecl
 	var volumes []corev1.Volume
 	args := []string{}
 	for _, secretEntry := range sa.Secrets {
+		if len(secretEntry.Name) == 0 {
```

Comment:
Suggest `if secretEntry.Name == "" {` to match line 61.

Reply:
Changed it!